packages/react-instantsearch/src/components/Breadcrumb.js

```diff
@@ -21,12 +21,14 @@ class Breadcrumb extends Component {
     items: itemsPropType,
     refine: PropTypes.func.isRequired,
     rootURL: PropTypes.string,
-    separator: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+    separator: PropTypes.node,
     translate: PropTypes.func.isRequired,
     className: PropTypes.string,
   };
 
   static defaultProps = {
+    rootURL: null,
+    separator: '>',
```

Comment:
Should it have spaces around?

Reply:
Don't think so, the default theme adds margin around the separator.

Reply:
Since it's `span` it will get read as `level>sublevel` and might not be what we want

Reply:
For screen reader you mean?

Reply:
yes